### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'int4WeightExtractionHalf'

### ISSUE THREAD

**Issue by JR-s763** [THUDM/ChatGLM-6B#1393]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Is this a model check point for proper loading or an int4 issue, has anyone encountered it?

### Expected Behavior

none

### Steps To Reproduce

none

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

### Anything else?

none

**Comment by eric-0x72** [comment]
any solution?

**Comment by JR-s763** [comment]
Yes,make sure you have executed "pip install -r requirements.txt"

**Comment by eric-0x72** [comment]
pip install  cpm_kernels
